realsense2_camera/src/t265_realsense_node.cpp

```diff
@@ -7,11 +7,13 @@ T265RealsenseNode::T265RealsenseNode(ros::NodeHandle& nodeHandle,
                                      rs2::device dev,
                                      const std::string& serial_no) : 
                                      BaseRealSenseNode(nodeHandle, privateNodeHandle, dev, serial_no),
+                                     _pose_sensor(dev.first<rs2::pose_sensor>()),
                                      _wo_snr(dev.first<rs2::wheel_odometer>()),
                                      _use_odom_in(false) 
                                      {
                                          _monitor_options = {RS2_OPTION_ASIC_TEMPERATURE, RS2_OPTION_MOTION_MODULE_TEMPERATURE};
                                          initializeOdometryInput();
+                                         setupMapReutilization();

```

Comment:
we call this function to initialize the map saving service, and to load the input map (if one is provided)